project-base/config/parameters_test.yml.dist

```diff
@@ -14,3 +14,5 @@ parameters:
     test_mailer_password: ~
 
     shopsys.content_dir_name: 'content-test'
+
+    env(ELASTIC_SEARCH_INDEX_PREFIX): 'test'
```

Comment:
Unfortunately, this does not work as a prefix... 
when I set environment variable ELASTIC_SEARCH_INDEX_PREFIX (in docker-compose file for example), then running tests will create index with the same name as production/development environment does

Reply:
This is really weird. This line is specifying the default value which might be overriden by value from ENV (docker-compose.yml).

I do not know about aby other solution except use for testing purposes second ENV (e.g. `TEST_ELASTIC_SEARCH_INDEX_PREFIX`). What do you think?

Reply:
it's moved from parameters into services